legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -667,13 +697,6 @@ def get_filing_types(business: Business, filing_json: dict):  # pylint: disable=
                 filing_type_code = Filing.FILINGS.get(k, {}).get('codes', {}).get(legal_type)
                 priority = priority_flag
 
-                # ToDo: bcgov/entity 12684, update the dissolution related code to support sub types.
-                # Check if it is administrative dissolution, then pass NOFEE code
-                if k == 'dissolution':
-                    dissolution_type = filing_json['filing'][k].get('dissolutionType')
-                    if dissolution_type == 'administrative':
-                        filing_type_code = 'NOFEE'
```

Comment:
I dont agree with this hardcoding that Ritvick made as part of the admin dissolution. I dint want to change it as part of this ticket. I will create a techdebt ticket to structure the dissolution metadata more effectively